common/components/SendButtonFactory/OnlineSend.tsx

```diff
@@ -24,7 +24,6 @@ interface DispatchProps {
 }
 
 interface OwnProps {
-  Modal: typeof ConfirmationModal;
```

Comment:
Removing this prop breaks #940 as a custom modal implementation can not be passed down to `OnlineSend` from `SendButtonFactory`